dot/types/parachain.go

```diff
@@ -0,0 +1,388 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package types
+
+import (
+	"fmt"
+
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/lib/crypto"
+	"github.com/ChainSafe/gossamer/lib/crypto/sr25519"
+	"github.com/ChainSafe/gossamer/pkg/scale"
+	"github.com/centrifuge/go-substrate-rpc-client/v4/types"
+)
+
+// GroupRotationInfo represents the group rotation info
```

Comment:
This is not informative enough. I know that probably rust also does not provide a great comment. 
It is unclear here what a group is made up of (validators?).